app/assets/javascripts/oxalis/model/actions/skeletontracing_actions.js

```diff
@@ -152,6 +161,25 @@ export const deleteNodeAction = (
   timestamp,
 });
 
+export const deleteNodeWithConfirmAction = (
+  nodeId?: number,
+  treeId?: number,
+): DeleteNodeActionType | NoActionType => {
+  const state = Store.getState();
+  return getActiveNode(state.tracing)
+    .map(activeNode => {
+      nodeId = nodeId != null ? nodeId : activeNode.id;
+      if (state.task != null && nodeId === 1) {
+        // Let the user confirm the deletion of the initial node (node with id 1) of a task
+        if (!Window.confirm("Do you really want to delete the initial node?")) {
```

Comment:
I wonder if we could use a nice and shiny Antd `confirm` here? ✨ https://ant.design/components/modal/#components-modal-demo-confirm
Also, we we should store all messages in the `messages.js`.